modules/rules_sh/0.3.0/patches/fix-version.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
+index 1610bed..b346764 100644
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -1,11 +1,15 @@
+ module(
+     name = "rules_sh",
+-    version = "0.2.0",
++    version = "0.3.0",
+     compatibility_level = 0,
+-    toolchains_to_register = ["@local_posix_config//:local_posix_toolchain"],
+ )
++
+ bazel_dep(name = "bazel_skylib", version = "1.0.3")
+ bazel_dep(name = "platforms", version = "0.0.4")
+ 
+-sh_configure = use_extension("@rules_sh//bzlmod:extensions.bzl", "sh_configure")
++sh_configure = use_extension("//bzlmod:extensions.bzl", "sh_configure")
++
+ use_repo(sh_configure, "local_posix_config")
++use_repo(sh_configure, "rules_sh_shim_exe")
++
++register_toolchains("@local_posix_config//:local_posix_toolchain")
+diff --git a/bzlmod/extensions.bzl b/bzlmod/extensions.bzl
+index d54f5c2..90a7f42 100644
+--- a/bzlmod/extensions.bzl
++++ b/bzlmod/extensions.bzl
+@@ -1,6 +1,14 @@
+ load("//sh:posix.bzl", "sh_posix_configure")
++load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+ 
+ def _sh_configure_impl(ctx):
+     sh_posix_configure(register = False)
++    http_file(
++        name = "rules_sh_shim_exe",
++        sha256 = "cb440b8a08a2095a59666a859b35aa5a1524b140b909ecc760f38f3baccf80e6",
++        urls = ["https://github.com/ScoopInstaller/Shim/releases/download/v1.0.1/shim.exe"],
++        downloaded_file_path = "shim.exe",
++        executable = True,
++    )
+ 
+ sh_configure = module_extension(implementation = _sh_configure_impl)
+diff --git a/tests/BUILD.bazel b/tests/BUILD.bazel
+deleted file mode 100644
+index aed19e4..0000000
+--- a/tests/BUILD.bazel
```

Comment:
You should be able to remove this patch and instead add `-@rules_sh//tests/...` to the target list in the presubmit yaml to disable the tests.

Reply:
Yes, that's actually how I tested, but the `bcr_validation.py` script returned a warning:
```
BcrValidationResult.NEED_BCR_MAINTAINER_REVIEW: The presubmit.yml file of rules_sh@0.3.0 doesn't match its previous version rules_sh@0.2.0, the following presubmit.yml file change should be reviewed by a BCR maintainer.
```
and I wasn't sure what this meant and if it would complicate things. But let's try...

Reply:
It just means I need to push a button to confirm you aren't mining Bitcoin now ;⁠-⁠)

Reply:
OK, I see. CI is paused and waiting for a maintainer... ;-)

Reply:
I think you need to turn
```
- '@rules_sh//... -@rules//sh/tests/...`
```
into 
```
- '@rules_sh//...'
- '-@rules//sh/tests/...'
```

Reply:
Ah yes. Done.